docs/ssss.md

```diff
@@ -1,3 +1,94 @@
+this approach **has some disadvantages**:
+
+1. It requires payment of a gas fee.
+2. It is dependent on the chosen mainnet's performance, security, reliability,
+   usability, political risks, etc.
+3. It is limited to the ecosystem of the chosen mainnet and cannot easily
+   interact with other mainnets.
+
+Simperby solves these problems by providing a standalone blockchain for a DAO.
+
+For more details, refer to [the comparisons (WIP)](./comparison.md)
+
+## Simperby
+
+Simperby is a blockchain engine that builds a standalone blockchain for a DAO.
+
+- Trustless
+- Distributed
+- Fault Tolerant
+- Lightweight
+- Highly Interoperable
+- Native Support for Multichain Treasuries
+
+For details on the core protocol, refer to the [protocol
+overview](./protocol_overview.md).
+
+### A Byzantine Fault Tolerant System
+
+- **Builds an independent layer-1 mainnet** like [Cosmos
+  SDK](https://v1.cosmos.network/sdk) or [Substrate](https://substrate.io/).
+- **One mainnet for one DAO**.
+- Has own **BFT consensus** called [Vetomint](./vetomint.md) optimized for
+  sporadic node operations.
+
+### Governance
+
+- A peer-to-peer voting mechanism (governance) is used to make decisions on the
+organization, which is then finalized by the consensus.
+- No tokens, no staking, no mining, no gas fee; Instead, **permissioned and
+explicitly nominated nodes**.
+- Has its own peer-to-peer communication channel for the organization.
+
+### Storage
+
+- Provides a distributed file system for the organization.
+- **Managed as a distributed Git repository**.
+- Blocks and transactions are stored as Git commits.
+- Governance and consensus proposals are stored as Git branches.
+
+### Interoperability
+
+- Native support for trustless message delivery that establishes **multichain
+  interoperability** directly.
+- A light client that verifies the finality of the chain without requiring the
+  full state or full block, sharing the same principle as [Cosmos
+  IBC](https://ibc.cosmos.network/).
+- Easy implementation over various existing mainnets, thanks to its super
+  lightweight architecture.
+- Effective control of treasuries, tokens, and other dapps by the organization
+  without requiring trusted third parties.
+- **Treasury contracts** for the supported mainnets are available out of the
+  box.
+
+### Sporadic & Lightweight Node Operation
+
+- Simperby's novel consensus protocol allows nodes to be operated sporadically.
+- **Blocks are produced on demand** by Simperby, only when there is a
+  governance-approved transaction.
+- The Simperby node is an extremely **lightweight CLI software** that **does not
+  run in the background**.
+- All operations are performed synchronously and explicitly.
+- Participants can act as client nodes, performing required operations,
+  broadcasting messages to the peer-to-peer network if needed, and returning
+  results immediately.
+- At least one working server node is required to receive broadcasted messages
```

Comment:
but has no authority -> but it has no authority

I think it would be better to explain a little more about what authority it has and doesn't have.

Reply:
fixed.